lib/systems/default.nix

```diff
@@ -68,6 +68,28 @@ rec {
       useAndroidPrebuilt = false;
       useiOSPrebuilt = false;
 
+      needsLibtoolUsrBinFileFix = isCross: with final;
```

Comment:
Suggestion: How about we always fix this to not needing to keep track of this and in projects? Because I don't think that anytime soon this is fixed *everywhere*.

Reply:
@SuperSandro2000, thank you so much for taking the time to review this PR (and my others too, but this one especially).  I really appreciate it; this is the PR I was afraid would get stuck in limbo forever.

> Suggestion: How about we always fix this to not needing to keep track of this and in projects?

I would prefer this too.  I was trying to minimize rebuilds, but in hindsight maintainability matters more.

If the fix is applied on all platforms then this definitely needs to go into staging.  The good news: if this goes into staging it can build off of #167071, which will make this PR much shorter.

I will incorporate your other feedback, switch branches, and remove the parts that are no longer necessary after #167071.

Again, thank you so much for wading into this complicated issue and taking the time to understand what's going on here!